[2022-07-23 09:21:04,049] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.create_snowflake_table manual__2022-07-23T09:20:46.397960+00:00 [queued]>
[2022-07-23 09:21:04,075] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.create_snowflake_table manual__2022-07-23T09:20:46.397960+00:00 [queued]>
[2022-07-23 09:21:04,076] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:21:04,076] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-23 09:21:04,076] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:21:04,116] {taskinstance.py:1397} INFO - Executing <Task(SnowflakeOperator): create_snowflake_table> on 2022-07-23 09:20:46.397960+00:00
[2022-07-23 09:21:04,133] {standard_task_runner.py:52} INFO - Started process 815 to run task
[2022-07-23 09:21:04,149] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_snowflake', 'create_snowflake_table', 'manual__2022-07-23T09:20:46.397960+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/localtosnowflake.py', '--cfg-path', '/tmp/tmp6x8lgx8j', '--error-file', '/tmp/tmpyr8iy9p8']
[2022-07-23 09:21:04,152] {standard_task_runner.py:80} INFO - Job 59: Subtask create_snowflake_table
[2022-07-23 09:21:04,365] {task_command.py:371} INFO - Running <TaskInstance: local_to_snowflake.create_snowflake_table manual__2022-07-23T09:20:46.397960+00:00 [running]> on host cbd64c50b1d6
[2022-07-23 09:21:04,502] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_snowflake
AIRFLOW_CTX_TASK_ID=create_snowflake_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-23T09:20:46.397960+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-23T09:20:46.397960+00:00
[2022-07-23 09:21:04,503] {snowflake.py:118} INFO - Executing: CREATE TABLE IF NOT EXISTS PUBLIC.YELLOW_TRIPDATA
(vendor_id int,
pickup_datetime timestamp,
dropoff_datetime timestamp,
passenger_count int,
trip_distance float,
rate_code_id int,
store_and_fwd_flag varchar,
pickup_location_id int,
dropoff_location_id int,
payment_type int,
fare_amount float,
extra float,
mta_tax float,
tip_amount float,
tolls_amount float,
improvement_surcharge float,
total_amount float,
congestion_surcharge float,
upload_date timestamp);
[2022-07-23 09:21:04,511] {base.py:68} INFO - Using connection ID 'my_snowflake_conn' for task execution.
[2022-07-23 09:21:04,516] {connection.py:262} INFO - Snowflake Connector for Python Version: 2.7.9, Python Version: 3.7.13, Platform: Linux-5.15.0-1014-azure-x86_64-with-debian-11.4
[2022-07-23 09:21:04,517] {connection.py:877} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-07-23 09:21:04,517] {connection.py:894} INFO - Setting use_openssl_only mode to False
[2022-07-23 09:21:06,787] {cursor.py:710} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-07-23 09:21:07,079] {cursor.py:734} INFO - query execution done
[2022-07-23 09:21:07,080] {snowflake.py:328} INFO - Running statement: CREATE TABLE IF NOT EXISTS PUBLIC.YELLOW_TRIPDATA
(vendor_id int,
pickup_datetime timestamp,
dropoff_datetime timestamp,
passenger_count int,
trip_distance float,
rate_code_id int,
store_and_fwd_flag varchar,
pickup_location_id int,
dropoff_location_id int,
payment_type int,
fare_amount float,
extra float,
mta_tax float,
tip_amount float,
tolls_amount float,
improvement_surcharge float,
total_amount float,
congestion_surcharge float,
upload_date timestamp);, parameters: None
[2022-07-23 09:21:07,080] {cursor.py:710} INFO - query: [CREATE TABLE IF NOT EXISTS PUBLIC.YELLOW_TRIPDATA (vendor_id int, pickup_datetim...]
[2022-07-23 09:21:07,486] {cursor.py:734} INFO - query execution done
[2022-07-23 09:21:07,486] {snowflake.py:338} INFO - Statement execution info - {'status': 'Table YELLOW_TRIPDATA successfully created.'}
[2022-07-23 09:21:07,486] {snowflake.py:342} INFO - Rows affected: 1
[2022-07-23 09:21:07,486] {snowflake.py:343} INFO - Snowflake query id: 01a5cbb1-0000-da7c-0000-22590004fe46
[2022-07-23 09:21:07,487] {connection.py:507} INFO - closed
[2022-07-23 09:21:07,716] {connection.py:510} INFO - No async queries seem to be running, deleting session
[2022-07-23 09:21:08,110] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=local_to_snowflake, task_id=create_snowflake_table, execution_date=20220723T092046, start_date=20220723T092104, end_date=20220723T092108
[2022-07-23 09:21:08,228] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-23 09:21:08,377] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
